src/frontend/src/icp/utils/cketh-transactions.utils.ts

```diff
@@ -40,14 +40,12 @@ export const mapCkEthereumTransaction = ({
 	Partial<Pick<Network, 'env'>>): IcTransactionUi => {
 	const { id, from, to, ...txRest } = mapIcrcTransaction({ transaction, identity });
 
-	const ckETHExplorerUrl =
-		IC_CKETH_LEDGER_CANISTER_ID === ledgerCanisterId && nonNullish(env)
-			? env === 'testnet' && nonNullish(STAGING_CKETH_LEDGER_CANISTER_ID)
-				? ledgerCanisterId !== STAGING_CKETH_LEDGER_CANISTER_ID
-					? `${CKETH_SEPOLIA_EXPLORER_URL}/${ledgerCanisterId}`
-					: CKETH_SEPOLIA_EXPLORER_URL
-				: CKETH_EXPLORER_URL
-			: undefined;
+	const ckETHExplorerUrl = nonNullish(env)
```

Comment:
What's `+`? If the goal is to compose a string, can we use string literals similarly as in the rest of the codebase?

Reply:
something seemed off when I wrote it... that was it! thank you!